src/goose/client.clj

```diff
@@ -45,3 +60,11 @@
   `args` are variadic."
   [opts sec execute-fn-sym & args]
   (enqueue opts (u/add-sec sec) execute-fn-sym args))
+
+(defn perform-every
+  "Schedule a function for periodic execution.
+  `cron-schedule` should be a string in the UNIX cron format.
+  `execute-fn-sym` should be a fully-qualified function symbol.
+  `args` are variadic. "
+  [opts cron-schedule execute-fn-sym & args]
```

Comment:
We can add another arg defining how many times a cron should run. 0/-1 can mean infinity.
This looks quite simple to be implemented. If scope increases, we can create another feature for this.

Reply:
Created an issue to track this: https://github.com/nilenso/goose/issues/75